djstripe/settings.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+"""Settings related to the djstripe app."""
```

Comment:
Full module docstring please

Authors are me, pydanny, lskillen, and chrissmejia
https://github.com/kavdev/dj-stripe/blame/%23162-api-updates-through_2015-07-28/djstripe/settings.py